migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql

```diff
@@ -0,0 +1,50 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- IDEMPIERE-2955 Adding Attribute set type field (M_AttributeSet_Type), support DMS/MM
+ Dec 2, 2015 11:53:13 AM IST
```

Comment:
error:
SP2-0734: unknown command beginning "Dec 2, 201..." - rest of line ignored.

Reply:
@globalqss No oracle DB server, so will install, test script and then do next commit. This way is too costly:)

Reply:
I'm testing locally in an docker oracle - this seems the last one - the line starting with "Dec 2" must be preceded by --

Probably you deleted the comment mark inadvertently

Reply:
@globalqss I just fixed that deleted comment mark.  I identified that but waiting for oracle setup done. Please test now. still I have not oracle DB ready but as you have tested, let's move this ticket.